images/huggingface-model-downloader/download.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+huggingface-cli download --local-dir $MODEL_DIR $MODEL_REPO
```

Comment:
As a follow up, we may need to do some smarter logic that ensures only required files get downloaded. For example, some repos have both safetensors and pytorch. I can tackle this.

Reply:
#274 Added